app/controllers/api/physical_servers_controller.rb

```diff
@@ -68,10 +70,6 @@ def change_resource_state(state, type, id)
       end
     end
 
-    def server_ident(server)
```

Comment:
Was there a reason this was moved? I think it would be better to leave it here if it is not essential to fixing the bug

Reply:
Well, ok. it was just for code organization reason